dat/missions/zalek/neburesearch/neburesearch_00.lua

```diff
@@ -0,0 +1,221 @@
+--[[
+   
+   Mission: Novice Nebula Research
+   
+   Description: A Za'lek student asks the player to help him with his reasearch. The player has to visit Doeston and Arandon.
+   Some minor complications on the way have to be expected.
+   
+   Difficulty: Easy
+
+]]--
+
+include "dat/scripts/numstring.lua"
+
+
+t_sys = {}
+t_sys[1] = "Doeston"
+t_sys[2] = "Arandon"
+
+-- localization stuff, translators would work here
+bar_desc = _("You see a young scientist talking with some pilots, apparently without success.")
+mtitle = {}
+mtitle[1] = _("Novice Nebula Research")
+misn_reward = _("%s credits and the gratitude of a student")
+mdesc = {}
+mdesc[1] = _("Go to the %s system.")
+title = {}
+title[1] = _("Bar")
+title[2] = _("Please...")
+title[3] = _("What a mess...")
+title[4] = _("Ready")
+title[5] = _("Technical problems")
+title[6] = _("Shadows in the darkness")
+title[7] = _("Mission Success")
+text = {}
+text[1] = _([["Hello Captain! You are a pilot, right? For my project I require a ship heading to the Sol Nebula. Certainly you must be interested in the proposal of researching the phenomenon that cut us off from mankind's patrimony.
+    As this is the point where any other pilots I asked backed out I should start with my problem; due to some unfortunate circumstances the payment for this mission will be only 50000 credits. But rest assured, you will be mentioned in the acknowledgment section of my next paper!"]])
```

Comment:
Numbers should be written with commas, so this should be "50,000 credits". Or better yet, use the creditstring() function from numstring.lua.

🕵️